Gems/Atom/RPI/Code/Source/RPI.Public/Culling.cpp

```diff
@@ -168,7 +168,10 @@ namespace AZ
             return worklistData;
         }
 
-        constexpr size_t WorkListCapacity = 5;
+        // Any value larger than 9 triggers a crash the job creation following the message:
+        // "Allocation size (560) is too big (max: 512) for pools!"
+        // (each additional count here increases the allocation size by 48, value of 9 makes the size exactly 512)
+        constexpr size_t WorkListCapacity = 9;
```

Comment:
I'm curious what impact it would have to go in the other direction, but with the taskgraph. A WorkListCapacity of 1 +TG. The TaskGraph is supposed to have lower overhead, and the purpose of accumulating nodes into a worklist is to avoid job system overhead. I would expect being more distributed to help, especially on a ThreadRipper.

In my experiments with batching up draw call creation, I was able to achieve the best utilization with smaller batches using the TaskGraph, otherwise several threads/cores were just idling.